go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/spf13/cobra v1.7.0
 	github.com/spf13/pflag v1.0.5
 	github.com/submariner-io/admiral v0.16.7
-	github.com/submariner-io/cloud-prepare v0.16.7
+	github.com/submariner-io/cloud-prepare v0.16.8-0.20240926122845-cd0adcc65c05
```

Comment:
I can’t access GH over SSH at the airport, so I can’t do it myself — this needs to be bumped to the commit hash as merged:

```suggestion
	github.com/submariner-io/cloud-prepare v0.16.8-0.20240926143602-f935ffc6c0e6
```